test/manager/ansible/__snapshots__/extract.spec.js.snap

```diff
@@ -9,6 +9,7 @@ Array [
     "currentValue": undefined,
     "datasource": "docker",
     "depName": "busybox",
+    "depType": "stage",
```

Comment:
`depType` shouldn't apply to ansible - only to `dockerfile`

Reply:
done